arches/app/utils/decorators.py

```diff
@@ -59,7 +59,7 @@ def new_func(*args, **kwargs):
     return new_func
 
 
-def group_required(*group_names):
+def group_required(*group_names, raise_exception=False):
```

Comment:
what's up with this change?

Reply:
Whoops, good catch, this shouldn't have snuck in when I rebased on latest dev/7.6